docs/it/README.md

```diff
@@ -118,12 +118,14 @@ Poiché Google [alla fine eliminerà gradualmente](https://developer.chrome.com/
 1. Salva https://raw.githubusercontent.com/kudoai/chatgpt.js/main/chatgpt.js in una sottocartella (`lib` in questo esempio)
 
 2. Aggiungi la dichiarazione di esportazione ES6 alla fine di `lib/chatgpt.js`
+
```

Comment:
When adding newlines after numbered headings, the gap as rendered by Docsify shrinks for some reason. Can you edit the CSS to preserve the prev distance? It will even fix this bug's effect https://github.com/docsifyjs/docsify/issues/2057 (and I can mass-edit the other docs using regex in VSC)

Reply:
the autosave messed up, I've just noticed, lemme revert the changes

Reply:
@madkarmaa the newlines? keep them if you edit the CSS (or I can do after my tasks) because it fixes the indent bug's effects as I mentioned

Reply:
@madkarmaa also your comment said ' +formatting' so I thought you wanted them?

Reply:
I didn't check them, autosave usually removes tab spaces on blank lines, so I thought it did that until I checked, my bad

Reply:
idk what you're saying, what is the meaning of your " + formatting" in your comment

Reply:
....because you removed all your formatting changes, not tab spaces on blank lines

Reply:
as I said it's a mistake, I reverted it

Reply:
newlines got added by mistake, every time autosave modifies something in a .md file is because of tabulations on empty lines and so I thought about that case without checking, my bad

Reply:
not just translations. each time I save a file it gets formatted unless I tell vscode not to do it
this time, I forgot to disable it, and so once I saved it changed some formatting in the file
I thought it was nothing important, just like sometimes it _removes_ tabs where they're not needed (eg. empty lines)
so I didn't check but I should've

Reply:
because you noted you made formatting changes, so I don't get how it got added by mistake

Reply:
I saw the lines had changed but I thought it was the tabulations thing, so I also said "formatting" but it was a mistake

Reply:
ok i typed my last reply while your msg before that was invisible beacuse github didn't autoload it, now i got it